### PR TITLE
small documentation improvements

### PR DESCRIPTION
**Pull request by yromanyshyn**

### CODE REVIEW

**Review by ItayGabbay — APPROVED** [review]

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,530 @@
...
+    result = f"* :doc:`API Reference - {notebook_name} <{apipath}>`"
+    return result
+
+
+
+# TODO: Not sure if we need this
+
+# class AutoAutoSummary(Autosummary):
+
+#     option_spec = {
```
Comment (benisraeldan) [inline docs/source/conf.py:467]: you're right we don't need this, you can remove

File: docs/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,530 @@
...
+
+
+CURRENT_DIR = pathlib.Path(__file__).parent
+PROJECT_DIR = CURRENT_DIR.parent.parent
+
+
+sys.path.insert(0, str(PROJECT_DIR.absolute()))
+
+
+import deepchecks.version
```
Comment (benisraeldan) [inline docs/source/conf.py:30]: I can't import this locally, can you remove it from here, or add it to the package?

File: makefile [hunk 3]
```diff
@@ -17,14 +17,24 @@ REQUIRE = requirements.txt
...
+>>>>>>> main
 repo = pypi
 
 # System Envs
 BIN := $(ENV)/bin
+<<<<<<< HEAD
+pythonpath := PYTHONPATH=.
+=======
 pythonpath = PYTHONPATH=.
+>>>>>>> main
```
Comment (benisraeldan) [inline makefile]: fix merge conflicts

Comment (yromanyshyn) [reply makefile]: already fixed)

File: makefile [hunk 4]
```diff
@@ -17,14 +17,24 @@ REQUIRE = requirements.txt
 ext_py := $(shell which python3 || which python)
 
 # Override by putting in commandline python=XXX when needed.
+<<<<<<< HEAD
+python = $(shell echo ${ext_py} | rev | cut -d '/' -f 1 | rev)
+TESTDIR = $(shell realpath tests)
+ENV = $(shell realpath venv)
```
Comment (benisraeldan) [inline makefile:22]: this doesn't work locally on my machine

File: docs/source/conf.py (hunk 2 above)
Comment (yromanyshyn) [reply docs/source/conf.py:30]: do you mean that you get ImportError when trying to import `deepchecks.version`?

File: makefile (hunk 4 above)
Comment (yromanyshyn) [reply makefile:22]: do you use windows? (I pushed fix for this)

File: docs/source/conf.py (hunk 1 above)
Comment (yromanyshyn) [reply docs/source/conf.py]: removed

File: makefile (hunk 4 above)
Comment (benisraeldan) [reply makefile:22]: nope, I'm on mac

**Comment by benisraeldan** [comment]
right now we have duplicate notebooks, is there a way to reference them from outside the source directory? or should we move them to the source and keep them update and tested there?

**Comment by yromanyshyn** [comment]
>right now we have duplicate notebooks, is there a way to reference them from outside the source directory? or should we move them to the source and keep them update and tested there?

@benisraeldan 

It looks like it is not possible to reference documents outside of the source dir in the rst/ at least I was not able to achieve this

Think we need to move notebooks to the docs/source folder

File: makefile (hunk 4 above)
Comment (yromanyshyn) [reply makefile:22]: weird/ Did I understand you right that there is no `realpath` cmd in mac env?

Comment (yromanyshyn) [reply makefile:22]: if so, I think this cmd will install it - `brew install coreutils`

**Comment by yromanyshyn** [comment]
> >right now we have duplicate notebooks, is there a way to reference them from outside the source directory? or should we move them to the source and keep them update and tested there?
> 
> @benisraeldan
> 
> It looks like it is not possible to reference documents outside of the source dir in the rst/ at least I was not able to achieve this
> 
> Think we need to move notebooks to the docs/source folder

@ItayGabbay any thoughts regarding that?

File: docs/source/conf.py (hunk 2 above)
Comment (benisraeldan) [reply docs/source/conf.py:25]: yup

**Pull request merged**